Acrolinx.Sidebar/Util/Message/MessageType.cs

```diff
@@ -0,0 +1,12 @@
+﻿/* Copyright (c) 2018 Acrolinx GmbH */

```

Comment:
@mscc-sascha Use appropriate copyright header for new files as per guideline https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/formatting.md